### DISCUSSION TITLE
When to use Infolist?

### DISCUSSION THREAD

**Discussion by notwhoyouexpected** [filamentphp/filament#12516] [General]
Hi, i am not sure when i should use a Infolist? i currently rebuilding my admin panel from zero as it got really messy in the last year. Now i think what can i do better? and i never used infolist so could someone give me 1-2 examples when/where i should use it? For example i have a really long edit user page where a lot is not editable and the stuff which is editable only gets mostly edited when creating a user and after that there a some Actions which we use but we dont really edit users after creating, should i use here a infolist? if so, how do i edit the user then if i need to in some cases? 

Edit User page:
![image](https://github.com/filamentphp/filament/assets/166851117/e4578d0e-34a9-4cc9-9861-92b1c89d67fd)

thanks in advance for your help and time!

**Comment by leandrocfe** [comment]
An Infolist is a great way to present read-only information to users about a specific record. It offers a fully flexible layout, allowing you to customize how the data is displayed.

Take a look
https://demo.filamentphp.com/blog/posts/1
